### PR TITLE
Add k8s-tools 1.14.2 both Alpine and Debian.

### PR DESCRIPTION
**Pull request by bbetter173**

### CODE REVIEW

File: k8s-tools/1.14.2-stretch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM python:2-stretch
+
+RUN apt update && \
+  apt-get -y install ca-certificates curl openssl tar gnupg bash postgresql-client mysql-client grep expect && \
+  update-ca-certificates && \
+  curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.13.4/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
```
Comment (CerealBoy) [inline k8s-tools/1.14.2-stretch/Dockerfile]: I would imagine this version would also be bumped?

File: k8s-tools/1.14.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM python:2-alpine
+
+# For details around the reasoning behind the symbolic link with musl, please see:
+# https://stackoverflow.com/questions/34729748/installed-go-binary-not-found-in-path-on-alpine-linux-docker/35613430
+
+RUN apk update && \
+  apk add --no-cache ca-certificates curl openssl tar gnupg bash postgresql-client mysql-client grep busybox-extras xz && \
+  update-ca-certificates && \
+  curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.13.4/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl && \
```
Comment (CerealBoy) [inline k8s-tools/1.14.2/Dockerfile:9]: And here?

**Review by CerealBoy — COMMENTED** [review]
Was ``helm`` the only version being bumped here?

**Comment by bbetter173** [comment]
Bumped the kubectl version to 1.14.2 as well - helmfile was left on 0.45 due to the large number of changes between it and 0.83 and a (as usual) lack of time to review and test the various changes.

**Review by CerealBoy — APPROVED** [review]
👍

**Comment by CerealBoy** [comment]
@HeWhoWas Images have now been pushed and are available

**Pull request merged**